internal/operators/manager.go

```diff
@@ -120,10 +123,35 @@ func (mgr *Manager) GetRequirementsBreakdownForHostInCluster(ctx context.Context
 	return requirements, nil
 }
 
+func (mgr *Manager) getStorageOperator(cluster *models.Cluster) (api.StorageOperator, error) {
+	priorityOrder := []string{odf.Operator.Name, lvm.Operator.Name}
+	for _, operatorName := range priorityOrder {
+		for _, operator := range cluster.MonitoredOperators {
+			if operator.Name == operatorName {
+				o := mgr.olmOperators[operatorName]
+				storageOperator, ok := o.(api.StorageOperator)
+				if !ok {
+					mgr.log.Errorf("defined storage operator %s does not implement StorageOperator interface", operatorName)
```

Comment:
I don't think we should log here.
You'll definitely have monitored operators that are not storage operators, right?